examples/webapp-management/01-AppServicePlan.sh

```diff
@@ -49,21 +49,21 @@ whpInfo2=`az app service plan get --name $whpName2`
 [ $(echo $whpInfo | jq '.sku.capacity' --raw-output) -eq $capacity ]
 
 printf "\n6. Get All app service plans by resource group: %s" "$groupName"
-plansByGroup=`az app service plan get --group $groupName`
+plansByGroup=`az appservice plan ls --group $groupName`
 [  $plansByGroup == *"$whpName"* ]
 [  $plansByGroup == *"$whpName2"* ]
 
 printf "\n7. Get All app service plans by location: %s" "$location2"
-plansByLocation=`az app service plan get -l $location2`
+plansByLocation=`az appservice plan ls -l $location2`
 [  $plansByLocation == *"$whpName2"* ]
 
 printf "\n8. Get All app service plans in a subscription."
-plansInSubscription=`az app service plan get`
+plansInSubscription=`az appservice plan ls`
 [  $plansInSubscription == *"$whpName"* ]
 [  $plansInSubscription == *"$whpName2"* ]
 
 printf "\n9. Remove app service plan: %s." "$whpName"
-az app service plan remove -n $whpName -g $groupName
+az appservice plan rm -n $whpName -g $groupName
 
 printf "\n10. Remove app service plan: %s." "$whpName2"
-az app service plan remove -n $whpName2 -g $groupName
+az appservice plan rm -n $whpName2 -g $groupName
```

Comment:
The branding name is in fact two words: https://azure.microsoft.com/en-us/services/app-service/. 

 

Mohit

 

From: Jason R. Shaver [mailto:notifications@github.com] 
Sent: Tuesday, January 12, 2016 4:39 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Subject: Re: [azure-powershell] Clu (#1620)

 

In examples/webapp-management/01-AppServicePlan.sh <https://github.com/Azure/azure-powershell/pull/1620#discussion_r49537583> :

>  
>  printf "\n10. Remove app service plan: %s." "$whpName2"
> -az app service plan remove -n $whpName2 -g $groupName
> +az appservice plan rm -n $whpName2 -g $groupName

appservice (one word) right? 

—
Reply to this email directly or view it on GitHub <https://github.com/Azure/azure-powershell/pull/1620/files#r49537583> .  <https://github.com/notifications/beacon/AA_eUuZG7AaL0U4nSP9zj1gGY3SInYMZks5pZZQ5gaJpZM4HCe60.gif>